pywavefront/obj.py

```diff
@@ -157,53 +172,103 @@ def consume_faces(self):
 
         We always triangulate to make it simple
         """
+        # Figure out the format of the first vertex
+        # We assume every consecutive vertex has the same format
```

Comment:
Alternate wording, based on my reading of the code below:

`We raise an exception if any following vertex has a different format`

Reply:
fixed